en/docs/tutorials/configure-automatic-deployment.md

```diff
@@ -0,0 +1,100 @@
+
+1. In the Choreo Console, navigate back to the REST API component for which you enabled automatic deployment.
+
+2. To update the implementation, click **Overview** in the left pane and then click **Edit Code**. The Web Editor opens.
+
+3. In the Web Editor, make an edit to the REST API implementation. For example, click the last return statement, replace `Hello` in the statement with `Hi`, and click **Save**.
+
+    !!! info
+        To edit this value, double-click it.
+
+    ![Edit implementation](../assets/img/tutorials/deploy-automatically/edit-implementation.textClipping){.cInlineImage-full}
+
+4. Click **Sync with Choreo Upstream** in the Git Status Bar.
+
+    ![Sync changes to implementation](../assets/img/tutorials/deploy-automatically/sync-changes-to-implementation.png){.cInlineImage-full}
+
+    In the message that appears, click **Sync my changes with Choreo**.
+
+5. In the left pane, enter a commit message (e.g., `Implement REST API`) and click the tick.
+
+    ![Commit message](../assets/img/tutorials/rest-api/commit-message.png){.cInlineImage-full}
+
+    You will get a message prompting you to configure your `user.name` and `user.email`. To proceed, follow the steps given below:
+
+    1. In the message, click **Open Git Log**.
+   
+    2. Click **Terminal** to open the in-built terminal and issue the following commands in it.
+   
+        - `git config --global user.email "<YOUR_GITHUB_EMAIL_ADDRESS>"`
+        - `git config --global user.name "<YOUR_GITHUB_USERNAME"`
+
+6. To push the changes to the private GitHub repository maintained by Choreo, click **0↓ 1↑** in the Git Status Bar.
+
+    ![Push changes](../assets/img/tutorials/deploy-automatically/push-changes.png){.cInlineImage-full}
+
+    !!! info
+        This icon only appears after the Web Editor completes the committing process.
+
+7. Click **OK** in the message that appears.
+
+8. In the **Username** field that appears, enter your GitHub username and press Enter/Return.
+
+    ![Enter user name](../assets/img/tutorials/deploy-automatically/enter-username.png){.cInlineImage-half}
+
+9. In the **Password** field that appears, enter your personal access token for GitHub and press Enter/Return.
+
+10. Go back to the tab in which you are running the Choreo Console.
+
+11. In the left pane, click **Deploy**.
+
+12. Click the refresh icon next to **Build and Deployment History**.
+
+     ![Refresh build and deployment history](../assets/img/tutorials/deploy-automatically/refresh-build-and-deployment-history.png){.cInlineImage-threeQuarters}
+
+    The **Build and Deployment History** section will display a new record with the **Automatic** label as shown below for the automatic build you triggered with your last commit.
+
+    ![Automatic deployment record](../assets/img/tutorials/deploy-automatically/automatic-deployment-record.png){.cInlineImage-threeQuarters}
+
+    You can click on this record to view the commit that triggered it in GitHub.
```

Comment:
If you click on the record it will open the build info pane. If you click on the commit hash it will open the commit that triggered the build in github.